modules/nw-multus-gitops.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/networking-gitops.adoc
+
+[id="nw-multus-gitops_{context}"]
+= Multiple networks and GitOps
+
+Typically, managing an additional network is performed imperatively by editing the `cluster` object for the Cluster Network Operator with the `oc edit networks.operator.openshift.io cluster` command and adding a `.spec.additionalNetworks` field.
+
+To support GitOps, the best practice is to begin by creating a network attachment definition object instead of modifying the `cluster` object.
+
+== Limitations
+
+* After you add a network attachment definition and a pod uses the network attachment, modifying the network attachment definition does not trigger the pod to restart. As a result, the pod continues to run with the previous definition.
```

Comment:
I think this is true in general, though. A pod must be deleted and re-created to benefit from updates to an existing NAD.